stable/artifactory-ha/templates/artifactory-node-statefulset.yaml

```diff
@@ -270,7 +270,7 @@ spec:
         - name: HA_NODE_ID
           valueFrom:
             fieldRef:
-              fieldPath: metadata.name
+              fieldPath: metadata.uid
```

Comment:
I'm not sure if there is an issue with the HA Node ID changing every time a new pod comes up. 

I initially tried to trim the release name off the pod name but had no luck due to the pod name not being available when the helm renders the templates.

Reply:
It is a problem as this id is used to register the node in the cluster and each node consumes a single license. After each pod redeployment, you'll get a new node registered in the cluster, which will show up on the list of instances and will not get a license. 
Need to look at maybe using a template for this.

Reply:
Looks like sprig has a uuid function (https://masterminds.github.io/sprig/uuid.html) that might also work. This means you would get a new HA Node ID every time you release which is slightly better.

Reply:
Still - means every upgrade will force new IDs. We need to persist the IDs. One of the reasons for using a statefulset.

Reply:
Ok, I'll create an issue for this so it is documented for anyone who runs into it while we are trying to find as solution 😃 

This is not a big deal for us now that we know what was causing our nuget issues but had us scratching our heads for awhile.